custom/sensors/TalonEncoder.java

```diff
@@ -0,0 +1,89 @@
+package org.usfirst.frc4904.standard.custom.sensors;
+
+
+import edu.wpi.first.wpilibj.CANTalon;
+import edu.wpi.first.wpilibj.PIDSourceType;
+
+public class TalonEncoder implements CustomEncoder {
+	protected final CANTalon talon;
+	private PIDSourceType pidSource;
```

Comment:
Why are these private? Should they not be protected? I can't foresee anything extending this but the philosophy is don't make it private unless you specifically want to disallow access to extending classes.